lib/src/sql/operator.rs

```diff
@@ -82,6 +83,7 @@ impl fmt::Display for Operator {
 			Operator::Sub => write!(f, "-"),
 			Operator::Mul => write!(f, "*"),
 			Operator::Div => write!(f, "/"),
+			Operator::Pow => write!(f, "**"),
```

Comment:
This will match the `Operator::Mul => write!(f, "*"),` line first, so the `**` needs to come before the `*` operator